internal/cloudrunci/gcloud.go

```diff
@@ -37,12 +37,13 @@ func init() {
 }
 
 // gcloud provides a common mechanism for executing gcloud commands.
-// It will attempt to retry failed commands 3 times with 2 second wait intervals.
+// It will attempt to retry failed commands. Use gcloudWithoutRetry() for no retry.
 func gcloud(label string, cmd *exec.Cmd) ([]byte, error) {
 	var out []byte
 	var err error
 
-	success := testutil.RetryWithoutTest(3, 2*time.Second, func(r *testutil.R) {
+	maxAttempts := 5
+	success := testutil.RetryWithoutTest(maxAttempts, 2*time.Second, func(r *testutil.R) {
```

Comment:
It might be good to also increase the sleep time to better offset attempts.

Reply:
Are you aware of any particular time interval used in other repos?

Reply:
No. This kicks off Cloud Builds right? I just assume Cloud Builds take longer than other retry requests we do for testing, so if you kick off 11 builds and 1 fails because of the quota, it may keep retrying while the other 10 builds are still running.

Reply:
Cloud Build is just one of several gcloud operations taken. In this repo, Cloud Build is only used to build the container image.

What size interval do you think is needed? Is going up to 3 sufficient or do you think we need to create much more space for another build to complete?

Reply:
If it's just building the docker container then maybe this isn't a concern.